### ISSUE TITLE
Errors while using elevation raster as landcover

### ISSUE THREAD

**Issue by benrsellers** [DEQrmichie/heatsource-9#15]
Hello,

I have been able to run the model using landcover code polygons as the landcover input. However, when I use the same dataset but use a digital elevation model as the landcover input I get the following error. 

Error: Traceback (most recent call last):                                 ] 1%
  File "C:\Users\RIPHAWK\anaconda3\envs\py3810\lib\site-packages\heatsource9\BigRedButton.py", line 280, in run
    HSP.run()
  File "C:\Users\RIPHAWK\anaconda3\envs\py3810\lib\site-packages\heatsource9\BigRedButton.py", line 194, in run
    self.run_all(time, hour, minute, second, JD, JDC)
  File "C:\Users\RIPHAWK\anaconda3\envs\py3810\lib\site-packages\heatsource9\BigRedButton.py", line 264, in run_hs
    [x.CalcHeat(time, H, M, S, JD, JDC) for x in self.reachlist]
  File "C:\Users\RIPHAWK\anaconda3\envs\py3810\lib\site-packages\heatsource9\BigRedButton.py", line 264, in <listcomp>
    [x.CalcHeat(time, H, M, S, JD, JDC) for x in self.reachlist]
  File "src\heatsource9\Stream\StreamNode.pyx", line 378, in heatsource9.Stream.StreamNode.StreamNode.calc_heat_boundary_node
  File "src\heatsource9\Stream\PyHeatsource.pyx", line 822, in heatsource9.Stream.PyHeatsource.calc_heat_fluxes
  File "src\heatsource9\Stream\PyHeatsource.pyx", line 345, in heatsource9.Stream.PyHeatsource.get_solar_flux
TypeError: unsupported operand type(s) for -: 'int' and 'NoneType'

Has anyone come across this error? Any advice would be much appreciated

**Comment by coltenelkin** [comment]
Ben;
Had the same error when I was getting this running. A full explanation of what I was seeing and what I did to fix it is included in a closed issue available on this github page. A copy of the key change is below:
"Took a while but I realized that this issue is the same as the one raised and solved by dharp on May 18. I went into the ModelSetup.py file (line 1147) and found where node.topo_factor is called out erroneously and changed it to node.TopoFactor (which matches all of the other callouts in the scripts)."

Hope this helps.
